### ISSUE TITLE
copy_statement causes forced replacement of pipe

### ISSUE THREAD

**Issue by henriblancke** [chanzuckerberg/terraform-provider-snowflake#180]
After successful creation of a pipe, every subsequent plan wants to recreate the pipe because it thinks the `copy_statement` changed.

Example:
```
resource "snowflake_pipe" "pipe" {
        auto_ingest          = true
        comment              = "Ceci n'est pas une pipe"
      ~ copy_statement       = <<~EOT # forces replacement
            COPY INTO DATABASE.SCHEMA.TABLE (JSON)
            FROM @DATABASE.SCHEMA.STAGE
        EOT
        database             = "DATABASE"
      ~ id                   = "DATABASE|SCHEMA|PIPE" -> (known after apply)
        name                 = "PIPE"
      ~ notification_channel = "arn:aws:sqs" -> (known after apply)
      ~ owner                = "TERRAFORM" -> (known after apply)
        schema               = "SCHEMA"
    }
``` 

Nothing changed to the copy statement, yet it seems to detect a change every time. I tried hard coding the statement where I first used a template file, both seem to have the same issue.

Thanks for taking a look, let me know if I'm just simply missing something!

**Comment by henriblancke** [comment]
I narrowed it down to the `copy_statement` being multi-line.

**Comment by ryanking** [comment]
@henriblancke is this resolved?

**Comment by henriblancke** [comment]
Yep resolved in #184
